implementations/lighthouse/Dockerfile

```diff
@@ -1,17 +1,25 @@
-FROM php:8.1.21-cli
-
-WORKDIR /app
+FROM php:8.2.19-cli
 
+# See https://getcomposer.org.
 COPY --from=composer /usr/bin/composer /usr/bin/composer
 
-RUN apt-get update && \
-    apt-get install --yes \
+# Required to make the composer installations work.
+RUN apt-get update \
+    && apt-get install --yes \
         git \
         libzip-dev \
         zip \
     && docker-php-ext-install \
         zip \
     && rm -rf /var/lib/apt/lists/*
-RUN composer create-project laravel/laravel /app
-RUN composer require nuwave/lighthouse 6.14.0
+
+# Creating a project following https://laravel.com/docs/installation#creating-a-laravel-project.
+# See https://github.com/laravel/laravel/releases for new versions.
+WORKDIR /app
+RUN composer create-project laravel/laravel=11.1.1 /app
```

Comment:
Using a fixed version here should stabilise the traces in most cases. Other dependencies might still change them in rare cases, we can fix them when it comes up.